apps/condo/domains/ticket/components/BaseTicketForm/TicketDeadlineField.tsx

```diff
@@ -86,12 +86,14 @@ export const TicketDeadlineField = ({ initialValues, form }) => {
         } else {
             form.setFields([{ name: 'deadline', value: autoDeadlineValue }])
         }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can this rule disabling be avoided?
For example to add `autoDeadlineValue` to dependencies or by another way?

Reply:
No, we should leave dependencies unchanged.
This logic should only be done to set initial values